### ISSUE TITLE
[Question] Windows 10 lightspark testing in chromium

### ISSUE THREAD

**Issue by JeimeRhee** [lightspark/lightspark#518]
## In order to help us understand your issue, please follow the guide below

- You will be asked some questions and requested to provide some information, please read them **carefully** and answer honestly
- Put an `x` into all the boxes `[ ]` relevant to your issue (like this: `[x]`)
- Please fill in the sections relevant to your issue and delete the rest.
- Use the *Preview* tab to see what your issue will actually look like

---

### What is the purpose of your issue?
- [ ] Site Support Request: Lightspark does not work correctly on a specific website.
- [ ] Bug: Lightspark has the same problem across multiple websites or SWF files.
- [ ] Feature request: a request for new features or support for new platforms, *not sites or SWF files*.
- [X] Question, Comment, or Suggestion
- [ ] If none of the above apply, check this box.

---

### The following sections should be filled in based on the purpose listed above. Erase any section (the contents between triple ---) that does not apply.

--- 

#### Provide the Lightspark version

Output of ``lightspark -v``:

``` 
INFO: Lightspark version 0.8.2-git Copyright 2009-2013 Alessandro Pignotti and others
```

#### What operating system are you are running on?

- [X] Windows 7, 8 or 10
- [ ] Windows XP, Vista or older
- [ ] Linux: *<specify distro and version here, such as Ubuntu 18.04>*
- [ ] Mac OS X: *<specify version here, such as 10.8.1>*
- [ ] Other: *<specify OS and version here>*

#### How are you are running Lightspark?

- [X] Browser plugin for Chrome or Chromium: *<specify Chromium version here>*
- [ ] Browser plugin for Waterfox: *<specify Waterfox version here>*
- [ ] Browser plugin for other browser: *<specify browser and version here>*
- [ ] Lightspark executable on the command line

Identify the type of request:

- [ ] Support for a new OS: *<name the operating system here>*
- [X] Support for a new browser: *<Chromium 78>*
- [ ] Support for embedding in an application: *<name the application here>*

---

### If this issue is a *question, comment, or suggestion*:

Identify the subject of this issue:

- [X] The documentation of how to use Lightspark, including the wiki
- [X] The build instructions to compile Lightspark from source
- [ ] The project and how it is managed (people, process, and so on)
- [ ] Other subject matter not listed

---

### Description of your issue

I downloaded "Lightspark-0.8.2-git-win64.exe" and I installed it.

I want testing lightspark flash plugin in chromium browser. 

So I launched chromium with "--ppapi-flash-path C:\Program Files\Lightspark\nplightsparkplugin.dll"

But chromium can't display flash game

I try to using "C:\Program Files\Lightspark\lightspark.dll" and "C:\Program Files\Lightspark\PepperFlash\libpepflashplayer.dll", but still can't display flash game.

![image](https://user-images.githubusercontent.com/15035272/85656721-0cb5ce80-b6ec-11ea-8bba-5e9d4f98673a.png)

Does anyone can help me?

**Comment by dbluelle** [comment]
I can't test this because I don't have a Windows machine so take the following with a grain of salt:

- The correct library to use is libpepflashplayer.dll.
- You have to make sure the manifest.json file is in the same directory as libpepflashplayer.dll
- libpepflashplayer.dll depends on lightspark.dll, so make sure lightspark.dll is in a location where chromium can find it 
(try putting it in the same directory as libpepflashplayer.dll)

**Comment by huwdp** [comment]
I wanted to see if I could reproduce the issue so I went ahead and added added ``` --ppapi-flash-path="C:\Program Files\Lightspark\PepperFlash\libpepflashplayer.dll"``` to the end of the a chromium path and copied libspark.dll to the PepperFlash directory. The player crashes in the web browser while viewing viewing a swf in a web page. I can't find any errors within EventViewer to help with this

![image](https://user-images.githubusercontent.com/364976/85922248-d908af00-b879-11ea-9f52-156cbcef2b35.png)

**Comment by dbluelle** [comment]
Maybe the crash has the same source as #496

**Comment by huwdp** [comment]
I'll test out the new version of Lightspark tomorrow, see if this works

**Comment by huwdp** [comment]
I have not been able to use updated Lightspark version within Chromium. I have copied libspark to Pepperflash and have added the argument to chromium shortcut.

I have tried the links below:
https://condor.depaul.edu/sjost/hci430/flash-examples/Examples1/Examples1.htm
https://www.miniclip.com/games/commando-3/en/#t-c-f-C

**Comment by ThanoS-max** [comment]
Yes even I am experiencing some difficulties to able to use updated Lightspark version within Chromium. Can you give me updates regarding this please?

**Comment by huwdp** [comment]
FYI: The next versions of Firefox/Chrome will have flash support removed completely, so the plugins will not work anymore.

Pale Moon allows the use of NPAPI so you could try that. Also Lightspark can be used on Windows standalone if you were to use command line, I only use Lightspark from command line.

https://support.mozilla.org/en-US/kb/end-support-adobe-flash
https://www.blog.google/products/chrome/saying-goodbye-flash-chrome/

**Comment by dbluelle** [comment]
So far nobody could find out why it doesn't work, because there are no entries regarding lightspark or flash anywhere in the debug log, so there is no real progress on that issue.

**Comment by VladPutinEu** [comment]
Any progress? How do I install the plugin on Microsoft Edge (Chromium)?

**Comment by huwdp** [comment]
It's not possible in Microsoft Edge. Only a few websites browsers supports plugins now (https://github.com/lightspark/lightspark/wiki/Getting-Lightspark-up-and-running-in-Web-Browser)
